src/client/components/ErrorSummary.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { css, SerializedStyles } from '@emotion/react';
+import { error, space } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
+import { SvgAlertTriangle } from '@guardian/src-icons';
+import { size } from '@guardian/src-foundations/size';
+
+interface ErrorSummaryProps {
+  error: string;
+  context?: string;
+  cssOverrides?: SerializedStyles | SerializedStyles[];
```

Comment:
I've been removing this pattern when I see it.

Why have I been doing this?
2 reasons:

1. Allowing people to specify their own custom styles that overrides those of the component is a useful pattern for authors of shared UI libraries, like Source. It's useful because you _can't know_ all the possible use cases and don't want to block people from doing what they need. So you create the 'recommended' variations of a component but give users an escape hatch. In many cases though, these escape hatches end up being closed and new props being added to deal with them.

But here, where we control both the code inside the component and the code calling it, there's no need for this pattern. If there is a requirement for a new use case, then we can add a new prop or maybe restructure the component (or create a new one - whatever works best). We don't need a temporary escape hatch.

2. Why not just add it? Because your props go from a tightly controlled interface where anyone can read them and understand the possibilities, to _the entire css api_. Which is really big 😉

Reply:
Ah, I did not realise that! That's useful to know, I'll remove the `cssOverrides` pattern here.